bedrock/careers/templates/careers/home.html

```diff
@@ -10,368 +10,341 @@
-      <img class="mzp-c-billboard-image" src="{{ static('img/careers/beyond-the-browser.png') }}" srcset="{{ static('img/careers/beyond-the-browser-high-res.png') }} 1.5x" alt="Illustration of a group of people working together to connect abstract dots and lines" height="358" width="358" />
-    </div>
-    <div class="mzp-c-billboard-content">
-      <div class="mzp-c-billboard-content-container">
-        <div class="mzp-c-billboard-content-inner">
-          <h2 class="mzp-c-billboard-title">Beyond the browser</h2>
-          <p class="mzp-c-billboard-desc">
-            While our Firefox family of products works to keep everyone safer and smarter
-            online, our Mozilla initiatives focus on shaping the future of the internet.
-          </p>
-        </div>
-      </div>
+  <section class="c-careers-diversity c-careers-section">
+    <div class="mzp-l-content mzp-t-content-md">
+      <h2 class="c-careers-section-title">
+        Diversity and Inclusion
+      </h2>
+      <p>
+        At Mozilla, our employees, volunteers and community members span the
+        globe. We want to ensure that our organization is as diverse as the
+        communities we serve – and that our culture is inclusive and welcoming
+        to all.
+      </p>
+      <p>
+        We’ve made some progress that we’re proud of, and we also have a lot
+        more work to do.
+      </p>
+
+      <p class="c-careers-button-wrapper">
+        <a href="{{ url('careers.diversity') }}" data-cta-type="button" data-cta-name="careers diversity" class="mzp-c-button mzp-t-secondary">Read more</a>
+      </p>
     </div>
   </section>
-</section>
-
-<div class="mzp-l-content apply-now-wrapper">
-  <div class="centered">
-    <a class="mzp-c-button" href="{{ url('careers.listings') }}">Apply Now</a>
-  </div>
-</div>
-
-<section class="mzp-l-content centered locations" id="locations">
-  <div class="mzp-c-call-out">
-    <div class="mzp-l-content">
-      <h2 class="mzp-c-call-out-title">A global view</h2>
-      <p class="mzp-c-call-out-desc">
-        With offices all over the world, we consider the internet from multiple cultures
-        and contexts.
+
+  <section class="c-careers-locations c-careers-section mzp-t-background-secondary">
+    <div class="mzp-l-content mzp-t-content-md">
+      <h2 class="c-careers-section-title">
+        We’ll meet you where you are
+      </h2>
+
+      <p>Mozilla is a remote-first company. Relying on our years of experience
+        in working asynchronously around the world, we’re reimaging what work
+        can look like – balancing your life with the connection and community
+        essential to the products we’re building.</p>
+
+      <div class="mzp-c-video">
+        <iframe width="640" height="360" src="https://www.youtube-nocookie.com/embed/_EeFDE5WW0I" title="Mozillians, kids, and pets on Zoom" frameborder="0" allowfullscreen>
+          <p><a href="https://www.youtube.com/watch?v=_EeFDE5WW0I&t">Watch the video</a></p>
```

Comment:
From my understanding, since the video is embedded onto our site, we're adding `nocookie` so that way no cookies are traced when the video is played directly on our website. However, the anchor tag is a fallback incase the `iframe` doesn't work and takes the user to the official Youtube page for the video to play. 
cc: @craigcook @alexgibson, happy for y'alls input in case I missed anything with the explanation

Reply:
We use this pattern elsewhere so I think this is fine as-is. If someone blocks iframes then they might not entertain watching youtube either, but this is probably the best we can do in this instance.